NAMESPACE

```diff
@@ -1,2 +1,68 @@
 # Generated by roxygen2: do not edit by hand
 
+export(addResidualError)
+export(allocateTreatments)
+export(analyseData)
+export(analyzeData)
+export(analyzeRep)
+export(applyPredCode)
+export(calculateDiffsFromBase)
+export(calculateObsSummary)
+export(calculateSimTrialMeans)
+export(checkColNames)
+export(checkDropOutFun)
+export(checkMacroFormat)
+export(checkMicroFormat)
+export(checkSimAlpha)
+export(checkSymmetricPDMatrix)
+export(compileSummary)
+export(convertSASIncludeRows)
+export(convertToSASCode)
+export(createContinuousCovariates)
+export(createCovariates)
+export(createDirectories)
+export(createDiscreteCovariates)
+export(createDropout)
+export(createEmptyMicro)
+export(createExternalCovariates)
+export(createExternalParameters)
+export(createInterims)
+export(createMCAR)
+export(createNmMatrix)
+export(createNormalParameters)
+export(createParameters)
+export(createResponse)
+export(createResponseVariable)
+export(createTimeVaryingCovariates)
+export(createTreatments)
+export(createTvDefinition)
+export(ectdStop)
+export(generateData)
+export(getEctdColName)
+export(getEctdExternalPath)
+export(getEctdPossibleColNames)
+export(getReplicates)
+export(initialChar)
+export(interimAnalysis)
+export(macroEvaluation)
+export(matchEctdColNames)
+export(parseCharInput)

```

Comment:
I think we don't need to export **_ALL_** of the functions. parse functions for example, can stay as internal functions.

Reply:
I already put the parse functions back to the internal functions. Regarding the rest exported functions, should I also put the low-level generate data components of generateData functions back into the internal function?